packages/phoenix-evals/src/phoenix/evals/classify.py

```diff
@@ -88,9 +95,8 @@ def llm_classify(
     `provide_explanation=True`.
 
     Args:
-        dataframe (pandas.DataFrame): A pandas dataframe in which each row represents
-            a record to be classified. All template variable names must appear as column
-            names in the dataframe (extra columns unrelated to the template are permitted).
+        dataframe (Union[pd.DataFrame, pd.Series, List[Any], Tuple[Any]): A collection of data which
```

Comment:
might want to change the name of this argument and also add a `dataframe` keyword argument to preserve backwards compatibility